Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ARKit
+import ArcGISToolkit
+import ArcGIS
+
+open class ARExample: UIViewController {
+    
+    public let arView = ArcGISARView(frame: CGRect.zero)
+    
+    private let scene: AGSScene = {
```

Comment:
Now that I see this implemented as a property, I regret suggesting it. There's a downside to the property approach that I missed: what happens if the user assigns their own scene to the scene view? This scene is no longer the correct scene and will not be deallocated until the view is deallocated. Since there's no way to prevent user's from changing the scene (as long as `arView` is public), I think using a method was the right approach, just with a better name.

Edit: I originally commented this in the wrong place.

Reply:
Back to a method:  `private func makeStreetsScene() -> AGSScene`

I think that works better; it may be that we'll have several different scenes the user can choose from, so having a method with `makeXXX` works for me.